Rubberduck.Core/Navigation/CodeExplorer/CodeExplorerViewModel.cs

```diff
@@ -424,6 +426,8 @@ private void ParserState_ModuleStateChanged(object sender, ParseProgressEventArg
 
                     // Force a refresh. OnPropertyChanged("Projects") didn't work.
                     Projects = Projects;
```

Comment:
This doesn't look right, we're invoking the setter just for its side-effects. The setter logic could be extracted to another method, that could be invoked here? Normally a VM setter is no-op if `value` matches the backing field..

Reply:
It was previously set up like this. Working on extracting it for the experience.